tpm2/constants.go

```diff
@@ -223,6 +223,7 @@ const (
 	cmdCertify            tpmutil.Command = 0x00000148
 	cmdCertifyCreation    tpmutil.Command = 0x0000014A
 	cmdReadNV             tpmutil.Command = 0x0000014E
+	CmdPolicySecret       tpmutil.Command = 0x00000151
```

Comment:
This looks a bit awkward as the only exported cmd const. But I can't think of a much better way.
Could you add a comment explaining what this is and why it's exported?

Reply:
Added, note that this comment changes how gofmt formats the code, so many more lines are included in the diff than strcitly necessary